Dockerfile.stable

```diff
@@ -1,82 +1,82 @@
 FROM cfmeqe/sel_base_fc29
 
-#TODO: figure why user 0 was needed
-USER 0
+ENV SELENIUM_VERSION=3.141.59 \
+    SELENIUM_HOME=/home/selenium \
+    SELENIUM_PORT=4445 \
+    REVERSE_PROXY_PORT=4444 \
+    FIREFOX_VERSION=60.4.0esr \
+    GECKODRIVER_VERSION=v0.20.1 \
+    DEFAULT_PROFILE_NAME=mylovelyprofile \
+    DISPLAY=:99
 
-ENV SELENIUM_VERSION 3.14.0
-ENV FIREFOX_VERSION 60.4.0esr
-ENV GECKODRIVER_VERSION v0.20.1
+ENV SELENIUM_PATH=$SELENIUM_HOME/selenium-server/selenium-server-standalone.jar \
+    PATH=$SELENIUM_HOME/firefox:/opt/google/chrome:$PATH
+
+WORKDIR $SELENIUM_HOME
 
-# nginx port for selenium requests
-EXPOSE 4444
 # selenium server port
-EXPOSE 4445
+EXPOSE $SELENIUM_PORT
+
+# reverse proxy port
+EXPOSE $REVERSE_PROXY_PORT
 
 # vnc port
 EXPOSE 5999
```

Comment:
I like the pattern of capturing these in ENV, want to include this as well?

Reply:
Sure